src/main/java/seedu/duke/food/FoodList.java

```diff
@@ -0,0 +1,81 @@
+package seedu.duke.food;
+
+import java.util.ArrayList;
+
+public class FoodList {
+    public static final String LS = System.lineSeparator();
+    public static final String MESSAGE_FOOD_LEFT_COUNT = "There %s %d food left." + LS;
+    public static final String MESSAGE_FOOD_COUNT = "You have consumed %d food items:" + LS;
+    public static final String MESSAGE_FOOD_CLEAR = "All food items have been deleted.";
+    public static final String MESSAGE_EMPTY_FOOD_LIST = "No food item is found.";
+    public static final String MESSAGE_FOOD_DELETED = "You have removed the food item: " + LS + "\t%s" + LS;
+    public static final String MESSAGE_FOOD_ADDED = "A food item has been added:";
+    private ArrayList<Food> foodRecords = new ArrayList<>();
+
+    /**
+     * Returns food item in the food list.
+     *
+     * @param index The index of the food item
+     * @return the food item with the given index
+     */
+    public Food get(int index) {
+        return this.foodRecords.get(index);
+    }
+
+    /**
+     * Returns the size of the array list.
+     *
+     * @return the size of the array list attribute
+     */
+    public int size() {
+        return foodRecords.size();
+    }
+
+    /**
+     * Adds a food item into the food list.
+     *
+     * @param food The food class object to add
+     */
+    public void add(Food food) {
+        this.foodRecords.add(food);
+        System.out.println(MESSAGE_FOOD_ADDED);
+        System.out.println("\t" + food);
+    }
+
+    /**
+     * Prints the list of all the food items.
+     */
+    public void printFoodList() {
```

Comment:
Is it possible to format the list into a String to be returned from this method? This would help to separate UI (printing) from the data classes.

Reply:
Sure, I will try to do so!